README.md

```
@@ -80,7 +80,7 @@ $ dela mcp --init-cursor       # Cursor: ~/.cursor/mcp.json
 $ dela mcp --init-vscode       # VSCode: ~/.vscode/mcp.json
 $ dela mcp --init-codex        # OpenAI Codex: ~/.codex/config.toml
 $ dela mcp --init-gemini       # Gemini CLI: ~/.gemini/settings.json
-$ dela mcp --init-antigravity  # Antigravity: ~/.gemini/antigravity-ide/mcp_config.json
+$ dela mcp --init-antigravity  # Antigravity: ~/.gemini/config/mcp_config.json
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)